manual/cli.rst

```diff
@@ -2715,7 +2711,7 @@ These options are valid between :qpdf:ref:`--add-attachment` and ``--``.
 
 .. qpdf:option:: --mimetype=type/subtype
 
-   .. help: attachment mime type, e.g. application/pdf
+   .. help: attachment mime type (e.g. application/pdf)
```

Comment:
Unfortunately, I had to change this to remove the trailing closed parentheses for a technical reason as explained in the "Note: 2022-01-22" comment at the top of cli.rst. I originally had it in parentheses.

Reply:
Sorry, missed that one.

Reply:
Yeah, I hate to have some technical detail interfere with better text, but it was the only case, and I decided it wasn't worth working around.